fvm/evm/types/errors.go

```diff
@@ -25,10 +25,9 @@ var (
 	// TODO: we might consider this fatal
 	ErrInsufficientTotalSupply = errors.New("insufficient total supply")
 
-	// ErrBalanceConversion is returned conversion of balance has failed, usually
-	// is returned when the balance presented in attoflow has values that could
-	// be marginally lost on the conversion.
-	ErrBalanceConversion = errors.New("balance converion error")
+	// ErrWithdrawBalanceRounding is returned when withdraw call has a balance that could
+	// yeild to rounding error, i.e. the balance contains fractions smaller than 10^8 Flow (smallest unit allowed to transfer).
+	ErrWithdrawBalanceRounding = errors.New("withdraw failed! balance is susceptible to the rounding error")
```

Comment:
should we add what is the biggest precision we support in the error description so the user knows how to react

Reply:
hmm, not sure if I follow, we have 10^8 in the go doc?